modules/lti/src/main/resources/tools/manage/js/lti.manage.js

```diff
@@ -1112,6 +1112,22 @@ function personalEventEditable(id) {
             '  <i class="fa fa-times-circle"></i></button></div>';
   return str;
 }
+function getTooltip(details){
+    var status = getStatus(details);
+
+    if (status == "Processing"){
+        return "Being processed for publication";
```

Comment:
This status is whenever a workflow is running, and could be for edit/preview or for publication or even for re-publication (in OBS case). We don't have enough detail to say which, so just make it:

Processing: please check back later.